code/modules/projectiles/updated_projectiles/projectile.dm

```diff
@@ -595,8 +596,9 @@ Normal range for a defender's bullet resist should be something around 30-50. ~N
 
 //Deal with xeno bullets.
 /mob/living/carbon/xenomorph/bullet_act(obj/item/projectile/P)
-	if(!P || !istype(P))
-		return
+	if(!istype(P))
```

Comment:
Can this check realistically happen? Does checking for it every single time make sense? These are the two questions you need to answer before putting this here would make sense rather than not having it at all.

Reply:
It can't happen.

Reply:
If it is happening then it needs to be fixed and the check below will easily detect it

Reply:
It cannot. I removed it in favor of a big screaming feedback just in case.
I intend to remove it at some point if we never get one such error message.

Reply:
Then it should just be removed outright. We can also ask ninja if you have doubts, though.

Reply:
Any reason to urgently remove it right now instead of letting it be tested a few rounds before removal?

Reply:
Becaus you haven't proven a reason for it to be there - a necessesity for this kind of check.

Reply:
Do notice I'm not adding the check.

Reply:
You're changing the line.

Reply:
Alright, I'll fix that.